security/providers/oidc-common/src/main/java/io/helidon/security/providers/oidc/common/OidcUtil.java

```diff
@@ -69,12 +72,14 @@ static ClientBuilder clientBaseBuilder(String proxyProtocol, String proxyHost, i
 
     static WebClient.Builder webClientBaseBuilder(String proxyHost,
                                                   int proxyPort,
+                                                  boolean relativeUris,
                                                   Duration clientTimeout) {
         WebClient.Builder webClientBuilder = WebClient.builder()
                 .addService(WebClientTracing.create())
                 .addMediaSupport(JsonpSupport.create())
                 .connectTimeout(clientTimeout.toMillis(), TimeUnit.MILLISECONDS)
-                .readTimeout(clientTimeout.toMillis(), TimeUnit.MILLISECONDS);
+                .readTimeout(clientTimeout.toMillis(), TimeUnit.MILLISECONDS)
+                .config(Config.create(ConfigSources.create(Map.of("relative-uris", String.valueOf(relativeUris)))));
```

Comment:
I am thinking that more correct approach would be to create `relativeUris(boolean relativeUris)` method on the WebClient.Builder, rather than creating new config just to set this.

Reply:
Implemented and added unit test for the change.